src/node/in_memory_ext.rs

```diff
@@ -768,10 +774,10 @@ mod tests {
             .expect("failed reading timestamp");
 
         let new_timestamp = timestamp_before + 500;
-        node.set_next_block_timestamp(new_timestamp)
+        node.set_next_block_timestamp(new_timestamp.into())
             .expect("failed setting timestamp");
 
-        let result = node.set_next_block_timestamp(timestamp_before);
+        let result = node.set_next_block_timestamp(timestamp_before.into());
 
```

Comment:
can be simplified to set new_timestamp as `timestamp_before - 1`

Reply:
Well, yes, it can - but why remove existing tests?